tool/tsh/mfa.go

```diff
@@ -663,3 +669,15 @@ func showOTPQRCode(k *otp.Key) (cleanup func(), retErr error) {
 		}
 	}, nil
 }
+
+func deleteTouchIDCredentialIfApplicable(credentialID string) {
+	err := touchid.AttemptDeleteNonInteractive(credentialID)
+	if err != nil {
+		if errors.Is(err, &touchid.ErrAttemptFailed{}) {
+			// Nothing to do here, just proceed.
+			return
+		}
+		log.WithError(err).Errorf("Failed to delete credential: %s\n", credentialID)
+		return
+	}
```

Comment:
Suggestion: a bit less nesting, scope of err reduced:

```suggestion
	switch err := touchid.AttemptDeleteNonInteractive(credentialID); {
	case errors.Is(err, &touchid.ErrAttemptFailed{}):
		// Nothing to do here, just proceed.
	case err != nil:
		log.WithError(err).Errorf("Failed to delete credential: %s\n", credentialID)
	}
```

Reply:
applied